app/src/main/java/org/simple/clinic/editpatient/EditPatientEffectHandler.kt

```diff
@@ -228,6 +227,14 @@ class EditPatientEffectHandler @AssistedInject constructor(
         .toObservable()
   }
 
+  private fun deleteBangladeshNationalId(savePatientEffects: Observable<SavePatientEffect>): Observable<EditPatientEvent> {
+    return savePatientEffects
+        .filter { isBangladeshIdCleared(it) }
```

Comment:
Same comment as earlier about either making this a method reference or making it a method on the `SavePatientEffect` type.